ios/Framework/Core/Context/SessionContext.swift

```diff
@@ -217,6 +217,7 @@ import Foundation
 	}
 
 	public class func logout() {
+		SessionContext.currentContext?.removeStoredCredentials()
```

Comment:
@victorg1991 don't do this. When a user logs out, it may want to keep the credentials stored (she only finish the session, but she may want to start later a new session using stored credentials).
If a developer want to do both (logout+remove credentials), she has methods for that